test/topology/pytest.ini

```diff
@@ -4,3 +4,7 @@ asyncio_mode = auto
 log_cli = true
 log_format = %(asctime)s %(levelname)s %(message)s
 log_date_format = %Y-%m-%d %H:%M:%S
+
+markers =
+    medium: tests that take several minutes to run
```

Comment:
It seems that we need to filter markers from the test.py command line by the markers from pytest.ini, since different suites can have different sets of markers. Or we should declare a common set of markers for all Python tests that run from test.py.

Reply:
https://docs.pytest.org/en/7.1.x/example/markers.html#registering-markers

has the following note:

```
Note
It is recommended to explicitly register markers so that:
- There is one place in your test suite defining your markers
- Asking for existing markers via pytest --markers gives good output
- Typos in function markers are treated as an error if you use the --strict-markers option.
```


so it seems not required, only recommended, and if we don't use `--strict-markers` then it's fine not to register them.